data/cameras.xml

```diff
@@ -904,14 +907,16 @@
 	<Camera make="Canon" model="Canon EOS 7D" mode="sRaw1">
 		<ID make="Canon" model="EOS 7D">Canon EOS 7D</ID>
 		<Crop x="0" y="0" width="0" height="0"/>
-		<Sensor black="0" white="44200" iso_list="100 125 160 320 640 1250"/>
-		<Sensor black="0" white="54000"/>
```

Comment:
I'm looking at ISO1250 sample [`raw.pixls.us/Canon/EOS 7D/RAW_CANON_EOS_7D-mraw.CR2` ](https://raw.pixls.us/data/Canon/EOS%207D/RAW_CANON_EOS_7D-mraw.CR2), and there is no way it should have white level of just `9000` instead of `44200`.

Reply:
I'll multiply by 4 and update them.

Reply:
I'll also shoot a few of the subsampled raws and import them to check and make sure the values make sense.

Reply:
I shot some mRaw and sRaw.  The 0 to 44000 matched the embedded jpegs best, so I'll set it back to that.